docs/manage-your-account/projects-and-env.mdx

```diff
@@ -0,0 +1,156 @@
+
+### Creating a new project
+
+If your organization needs to deal with **multiple projects**, you can add this project directly within the Projects page.
+
+<video controls>
+	<source src={createProject} />
+</video>
+
+### Editing, Filtering and Deleting projects
+
+We also provide the functionality for **filtering** through projects, to provide you with a more concrete view, and of course,
+you also get the ability to **edit** and **delete** projects as you wish.
+
+<video controls>
+	<source src={editFilterDeleteProj} />
+</video>
+
+### Getting the project id
+
+To retrieve the project ID, click on the three dots and the key will be visible in the pop-ip, or you can consult
+[this guide](/api/examples/get-project-and-env) for details on obtaining both the project and environment ID's via the API.
+
+![Getting Project ID](/ui-videos/project-and-env/project-id.png)
+
+:::note
+You will need a project and environment ID for most API calls to Permit.
+:::
+
+### Managing access to a project
+
+If you want to dive deeper into understanding member roles, refer to the
+[Member Management](/manage-your-account/workspace-settings#member-management) guide.
+
+<video controls>
+	<source src={manageAccessProj} />
+</video>
+
+## Environments
+
+Within your project, you create environments that suit your needs e.g. **dev**, **staging**, **production** etc.
+
+:::info
+Notice when you switch environment the **Policy Editor** will be updated to reflect the new environment.
+:::
+
+Every environment has a **secret key** that you use to check for permissions with the Permit's SDK and API.
+We offer simple functionality so that you can **easily copy your keys**, **rotate** them if you want to maintain your
+security to your desired standard, **edit** the environment to you liking, **delete it** if need be, and most importantly **change
+environment** which is as simple as a click of a button.
+
+You can manage environments from the **[Project](https://app.permit.io/project-management)** page.
+
+![Environment Page](/ui-videos/project-and-env/manage-envs.png)
+
+### Creating a new Environment
+
+<video controls>
+	<source src={createEnvironment} />
+</video>
+
+### Getting the environment id
+
+To retrieve the environment ID, click on the three dots and the key will be visible in the pop-ip, or you can consult
+[this guide](/api/examples/get-project-and-env) for details on obtaining both the project and environment ID via the API.
+
+:::note
+You will need a project and environment ID for most API calls to Permit.
+:::
+
+![Get env id](/ui-videos/workspace-settings/getting-env-id.png)
+
+### Fetching and rotating the API key
+
+![Get env id](/ui-videos/workspace-settings/fetch-and-rotate-api-key.png)
+
+### Editing and deleting environments
+
+<video controls>
+	<source src={editEnvironment} />
+</video>
+
+![Get env id](/ui-videos/workspace-settings/delete-env.png)
+
+### Managing access to an environment
+
```

Comment:
I miss a sentence that explain why we need this access management and why is it stands for exactly